tachyon/math/polynomials/univariate/univariate_evaluations_ops.h

```diff
@@ -200,8 +211,13 @@ class UnivariateEvaluationsOp {
       // 0 / g(x)
       return self;
     }
-    // TODO(chokobole): Check division by zero polynomial.
-    // f(x) / 0 skips this for loop.
+    if (r_evaluations.empty()) {
+      // f(x) / 0
+      // TODO(chokobole): It should return std::nullopt.
+      // See https://github.com/kroma-network/tachyon/issues/76.
+      return self;
+    }
```

Comment:
I think it's better to check `r_evaluation.empty()` before `l_evaluation.empty()`.